model/clusters_mgmt/v1/aws_inquiries_resource.model

```diff
@@ -16,13 +16,18 @@ limitations under the License.
 
 // Manages the collection of aws inquiries.
 resource AWSInquiries {
-	// Reference to the resource that manages a collection of vpcs.
-	locator Vpcs {
-		target VpcsInquiry
-	}
+    // Reference to the resource that manages a collection of vpcs.
+    locator Vpcs {
+        target VpcsInquiry
+    }
 
-	// Reference to the resource that manages a collection of regions.
-	locator Regions {
-		target AvailableRegionsInquiry
-	}
+    // Reference to the resource that manages a collection of regions.
+    locator Regions {
+        target AvailableRegionsInquiry
+    }
+
+    // Reference to the resource that manages aws sts policies.
+    locator AWSStsPolicies {
```

Comment:
@jhernand  can you ptal please? This is the end point 

`ocm get /api/clusters_mgmt/v1/aws_inquiries/aws_sts_policies`  

If i generate the model it generates the files like "awssts_policies_builder" but it is detecting the end point properly. Is it ok to proceed?

Reply:
It is OK, yes. However I think you can remove the `aws_` prefix; it is redundant. I mean:

```
locator STSPolicies {
        ....
}
```

That will correspond to `/api/clusters_mgmt/v1/aws_inquiries/sts_policies`. That is cleaner in my opinion (and will force you to change the server, unfortunately).

Also take into account that for situations like this where you have names with multiple acronyms concatenated the name parser of the metamodel may make wrong guesses about what you mean. In those cases you may want to use the `@go`, `@http` and `@json` annotations (recently introduced) to tell the metamodel what names do you want exactly in the generated code. For example, If you still want to use the `AWS` prefix and you want to write the `STS` acronym uppercase (as it should be) then you could do this:

```
@go(name = "AWSSTSPolicies")
@http(name = "aws_sts_policies")
locator AWSSTSPolicies {
        ...
}
```